php/php.editor/src/org/netbeans/modules/php/editor/codegen/CGSGenerator.java

```diff
@@ -418,7 +418,7 @@ public void invoke() {
         Preferences preferences = null;
         Project project = FileOwnerQuery.getOwner(fo);
         if (project != null) {
-            preferences = ProjectUtils.getPreferences(project, CGSGenerator.class, false);
+            preferences = ProjectUtils.getPreferences(project, CGSGenerator.class, true);
```

Comment:
I would write the reason as a comment.

Reply:
Trying to remember why it is done so... no idea.

Reply:
Comment added.

Reply:
Thanks.